dax/controller/http/handler.go

```diff
@@ -194,6 +195,32 @@ type DatabasesRequest struct {
 	// DatabaseNames     dax.DatabaseNames     `json:"database-names"`
 }
 
+// handlePatchDatabaseOptions handles updates to database options.
+func (s *server) patchDatabaseOptions(w http.ResponseWriter, r *http.Request) {
+	// Decode request.
+	var req DatabaseOptionRequest
+	dec := json.NewDecoder(r.Body)
+	dec.DisallowUnknownFields()
+	if err := dec.Decode(&req); err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	if err := s.controller.SetDatabaseOption(r.Context(), req.QualifiedDatabaseID, req.Option, req.Value); err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+}
+
+// DatabaseOptionRequest represents a change to a database option. The thinking
+// is to only support changing one database option at a time to keep the
+// implementation sane. At time of writing, only WorkersMin is supported.
+type DatabaseOptionRequest struct {
+	QualifiedDatabaseID dax.QualifiedDatabaseID `json:"qdbid"`
```

Comment:
is there any way we could change this json tag to "qdbpd"? just to really stick it to dyslexic folks?

Reply:
i think i'm going to set up code generation to create these with random variable names for every instance.